### PR TITLE
feat(contributing): Product Improvement Program

### PR DESCRIPTION
**Pull request by rbioteau**
First draft of the Product Improvement Program messaging.

### CODE REVIEW

File: modules/contributing/pages/product-improvement-program.adoc [hunk 1]
```diff
@@ -0,0 +1,47 @@
+= Product Improvement Program
+
+== Introduction
+
+Bonita Runtime and Bonita Studio collects anonymous telemetry data that helps Bonitasoft teams in understanding how the product is being used and in deciding what to focus on next.
```
Comment (educhastenier) [inline modules/contributing/pages/product-improvement-program.adoc]:
```suggestion
Bonita Runtime and Bonita Studio collect anonymous telemetry data that help Bonitasoft teams in understanding how the product is being used and in deciding what to focus on next.
```

File: modules/contributing/pages/product-improvement-program.adoc [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+The data collected is minimal, statistical in nature and cannot be used to uniquely identify a user. Please see the next section to see what data is collected and sent. Access to collected data is strictly limited to the Bonitasoft team and not shared with 3rd parties.
+
+As a growing community, we greatly appreciate the usage data users send to us, as it is very valuable in helping us make Bonita Platform a better product for everyone! However, if you are concerned about sending usage stats, you can choose to disable telemetry as described xref:contributing:product-improvement-program.adoc#disable-telemetry[here].
+
+== What data is collected?
```
Comment (educhastenier) [inline modules/contributing/pages/product-improvement-program.adoc:13]:
```suggestion
== What data are collected?
```

File: modules/contributing/pages/product-improvement-program.adoc [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+=== Runtime
+
+The runtime sends the executed connectors/actor filters id along their input and output names. This allows us to understand which connectors are used and how they are configured.
```
Comment (educhastenier) [inline modules/contributing/pages/product-improvement-program.adoc]:
```suggestion
The runtime sends the executed connectors/actor filters id along with their input and output names. This allows us to understand which connectors are used and how they are configured.
```

File: modules/contributing/pages/product-improvement-program.adoc [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+[NOTE]
+====
+For Subscription editions only.
+====
+
+== Disable Runtime telemetry
+
+You can turn off telemetry on the Runtime by setting the env variable `BONITA_RUNTIME_DATA_COLLECTION_MODE=disabled` or the system property `bonita.runtime.data-collection.mode=disabled`.
+This property can also be set in the `bonita-platform-sp.properties` file.
```
Comment (educhastenier) [inline modules/contributing/pages/product-improvement-program.adoc]: ⚠️ 
```suggestion
This property can also be set in the `bonita-platform-sp-custom.properties` file.
```

**Comment by educhastenier** [comment]
@rbioteau you're missing the `:description:` field at the top of the page

File: modules/contributing/pages/product-improvement-program.adoc [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+== What data is collected?
+
+=== Runtime
+
+The runtime sends the executed connectors/actor filters id along with their input and output names. This allows us to understand which connectors are used and how they are configured.
+
+.Runtime telemetry data sample
+[source,json]
+----
+{
```
Comment (nchabanoles) [inline modules/contributing/pages/product-improvement-program.adoc:22]: Please add a note to reinforce the fact that we collect the 'keys' not the 'values'. If needed use a concrete example, i.e. 
If your connector is called with 
"URL":"https://mydomain.com/service" We will collect the text "URL" and not see the actual value you used.

File: modules/contributing/pages/product-improvement-program.adoc [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+  "definition_id": "a-definition-id",
+  "definition_version": "1.0",
+  "input_names": ["inputName1", "inputName2"],
+  "output_names": ["outputName1", "outputName2"]
+}
+----
+
+=== Studio
```
Comment (nchabanoles) [inline modules/contributing/pages/product-improvement-program.adoc:32]: Please add a JSON example corresponding to the textual description, e.g.
{
"extension-type": "connector"
"extension-id": "a-definition-id",
"extension-version":""
}

**Review by nchabanoles — CHANGES_REQUESTED** [review]
Please complete the information for transparency sake (see comments)

File: modules/contributing/pages/product-improvement-program.adoc (hunk 6 above)
Comment (rbioteau) [reply modules/contributing/pages/product-improvement-program.adoc:32]: @nchabanoles As we won't be able to use a POST request for now, do we still add an example of a GET request ?

**Pull request merged**